manifests/params.pp

```diff
@@ -349,7 +349,9 @@
       $php_version = $facts['os']['release']['major'] ? {
         '9'     => '7.0', # Debian Stretch
         '10'    => '7.3', # Debian Buster
+        '16.04' => '7.0', # Ubuntu Xenial Xerus
```

Comment:
Ubuntu 16.04 is EOL and unsupported. Please keep it out.

Reply:
Do you mind if I remove this commit and do a force push ?

Reply:
Not at all. In fact, that's what I do all the time.